### ISSUE TITLE
Logo not displaying?

### ISSUE THREAD

**Issue by mhfowler** [YunoHost-Apps/outline_ynh#68]
I'm running into an issue where the logo is not displaying. It cannot find the resource, and is showing the alt text instead, even after seeming to successfully upload a new logo file. 

![Screenshot from 2023-03-06 12-16-05](https://user-images.githubusercontent.com/1831536/223039001-cd8f6b27-59d6-4321-912b-61648ae09429.png)

I am able to successfully add and see images within documents, using the /image command. 

I noticed that the URLs are different for the logo vs for the images within documents. 

Within a document that URL for the image looks something like:
/api/attachments.redirect?id=ea4f1d92-7b33-4cdf-8ca3-445d47b2cd3c 

For the logo, the URL for the image looks something like:
https://minio.commoninternet.net/outlinestorage/public/434e5e5e-337a-433e-8371-c7cd1aba3e9c/ea4f1d92-7b33-4cdf-8ca3-445d47b2cd3c/solidarity.jpg

which, when followed, leads to a access denied message. 

Any ideas what the issue could be?

**Comment by Limezy** [comment]
Could you please retry with the latest version v0.72 ?
It gets rid of MinIO so these kind of issues should now be solved.

**Comment by Limezy** [comment]
Closing as I didn't get any answer after a few months. Feel free to re-open
